### ISSUE TITLE
Arguments to simulate should raise warning/error

### ISSUE THREAD

**Issue by 0u812** [sys-bio/roadrunner#130]
Via Matthias:
Example in van_der_pol.py
**A**] The end time in the first simulation is not used ! I think this is a bug with positional and keyword arguments. The first simulation should run until 100, but only runs until 5.
**B**] If arguments are not existing they should give a warning, i.e. check the keywords to simulate. For instance the varSteps is simulating but not setting varSteps. Only after setting variableStep it works => typos will result in no warnings, but wrong simulations
```
rr.simulate(start=0, end=100, varSteps=True, stiff=True, plot=True)
rr.simulate(0, 100, varSteps=True, stiff=True, plot=True)

# unused arguments should raise a warning !
rr.simulate(start=0, duration=100, variableStep=True, stiff=True, plot=True)
```

**Comment by 0u812** [comment]
Duplicate of #125

**Comment by 0u812** [comment]
Need to test this

**Comment by 0u812** [comment]
Continued in #180
